burotel/indico_burotel/client/js/util.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+/**
+ * This function instantiates a React.Component with a set of additional passed props.
+ *
+ * @param {React.Component} Component - the actual component that will be parametrized
+ * @param {Object|Function} extraProps - additional properties passed to the final component
+ */
+export function parametrize(Component, extraProps) {
+    return function ParametrizedComponent({...props}) {
```

Comment:
what about assigning a proper name to those functions, e.g. `Parametrized<OriginalComponentName>`

Reply:
That won't work because some components (functional ones) have no `displayName` and some are wrapped in `Connect`s, so you'd see `ParametrizedConnect`.

Reply:
hm isn't the function name exposed as well somewhere? (maybe check what the react devtools use to get the name for these components)